### PR TITLE
generate private key on testnet

### PR DESCRIPTION
**Pull request by tys-hiroshi**

### CODE REVIEW

File: mnemonic/mnemonic.py [hunk 1]
```diff
@@ -250,15 +250,18 @@ def to_seed(cls, mnemonic, passphrase=""):
         return stretched[:64]
 
     @classmethod
-    def to_hd_master_key(cls, seed):
+    def to_hd_master_key(cls, seed, network='main'):
```
Comment (prusnak) [inline mnemonic/mnemonic.py]: `def to_hd_master_key(cls, seed, testnet=False):`

File: mnemonic/mnemonic.py [hunk 2]
```diff
@@ -250,15 +250,18 @@ def to_seed(cls, mnemonic, passphrase=""):
...
         if len(seed) != 64:
             raise ValueError("Provided seed should have length of 64")
 
         # Compute HMAC-SHA512 of seed
         seed = hmac.new(b"Bitcoin seed", seed, digestmod=hashlib.sha512).digest()
 
         # Serialization format can be found at: https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki#Serialization_format
-        xprv = b"\x04\x88\xad\xe4"  # Version for private mainnet
+        if network == 'main':
```
Comment (prusnak) [inline mnemonic/mnemonic.py]: if testnet ... elif: ...

File: mnemonic/mnemonic.py (hunk 1 above)
Comment (tys-hiroshi) [reply mnemonic/mnemonic.py]: @prusnak Thank you for your review.
I modified.

File: mnemonic/mnemonic.py (hunk 2 above)
Comment (tys-hiroshi) [reply mnemonic/mnemonic.py]: @prusnak Thank you for your review.
I modified.

**Comment by prusnak** [comment]
Merged, thanks for the contribution!

**Pull request merged**